mkdocs.yml

```diff
@@ -4,8 +4,9 @@ repo_name: zarr-developers/VirtualiZarr
 repo_url: https://github.com/zarr-developers/VirtualiZarr
 site_description: Cloud-Optimize your Scientific Data as Virtual Zarr stores, using Xarray syntax.
 site_author: VirtualiZarr developers
-site_url: https://virtualizarr.readthedocs.io/en/stable/index.html
+site_url: https://virtualizarr.readthedocs.io/en/latest/
```

Comment:
Any reason you can't just do the following to avoid adding the `prep-site-url` command in `pyproject.toml`?

```
site_url: !ENV [READTHEDOCS_CANONICAL_URL, 'https://virtualizarr.readthedocs.io/en/latest/']
```

Reference: https://www.mkdocs.org/user-guide/configuration/#environment-variables

Reply:
Amazing, thank you @chuckwondo! I tried searching for this solution but did not find that page. I applied your suggestion in [`b584a25` (#651)](https://github.com/zarr-developers/VirtualiZarr/pull/651/commits/b584a251ab97c6cd8d12b77f1ff183cbbe69ad28).